user-docs/pupil-capture.md

```diff
@@ -209,67 +224,190 @@ If you open up a session folder you will see a collection of video(s) and data f
 
 Click on the selector "Open Plugin" and select your plugin.
 
-### Pupil Groups
+#### Third-party plugins
+
+Third-party plugins can be loaded easily as well. In order for them to appear in
+the plugin list they need to be copied to the correct location, the `plugins`
+folder within your `capture_settings`.
+
+### Fixation Detection
+
+The built-in fixation detectors are dispersion-duration based.
+This means that if the pupil does not move more than a given distance (dispersion)
+in a given time period (duration) the plugin will classify the pupil positions
+during this time range as a fixation.
```

Comment:
Consider: _Dispersion_ is measured as the distance between pupil positions. _Duration_ is a specified period of time. The fixation detector plugin will classify pupil positions as fixations if they are within the dispersion for the duration of time specified.